Test/E2ETests/E2ETests/docker-compose452AppOn462.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+
+services:
+    e2etestwebapi:
+        build:
+            context: ./../TestApps/Net452/E2ETestWebApi/Publish
+            dockerfile: DockerFile462
+        depends_on:
+            - ingestionservice
+            
+    e2etestwebapp:
+        build: 
+            context: ./../TestApps/Net452/E2ETestApp/Publish
+            dockerfile: DockerFile462
+        depends_on:
+            - e2etestwebapi
+            - sql-server
+
+    ingestionservice:
+        build:
+            context: ./../IngestionService/Publish
+            dockerfile: DockerFile              
+
+    azureemulator:
+        build:
+            context: ./AzureEmulator
+            dockerfile: DockerFile        
+
+    sql-server:
+        image: microsoft/mssql-server-windows-express
+        environment:            
+            sa_password: MSDNm4g4z!n4
```

Comment:
Something to consider, but is probably a low priority.

Instead of hardcoding the password, I think you could make this value an argument in the yml. And then when the yml is executed in the ClassInitialize, it could read the value from a constant. 

My concern is if this ever needs to be changed, will have to search for all uses of this string.
That may not ever be an issue, but I wanted to bring it up for consideration. :)

Reply:
its a valid point though we dont need to change password.
I am moving EVERY constants off to a new Connstants project. Will address this then.